core/chaincode/exectransaction.go

```diff
@@ -139,6 +140,9 @@ func ExecuteTransactions(ctxt context.Context, cname ChainName, xacts []*pb.Tran
 	txerrs = make([]error, len(xacts))
 	for i, t := range xacts {
 		_, txerrs[i] = Execute(ctxt, chain, t)
+		if txerrs[i] != nil {
+			sendTxRejectedEvent(xacts[i].Uuid, txerrs[i].Error())
```

Comment:
@gabre this PR is to be taken in conjunction with #1719 ? If so the event _has_ to be emitted in execute and cannot be in commit correct (only successful txs are available in commit) ?